test/processors/text_processor_test.rb

```diff
@@ -44,12 +44,11 @@ class TextProcessorTest < ActiveSupport::TestCase
         audio_monster.expect(:create_temp_file, Tempfile.new('test'), [String, false])
         audio_monster.expect(:info_for, { format: 'text' }, [String])
       end
-
-      stub_request(:post, "http://api.opencalais.com/tag/rs/enrich").
-        with(headers: {'Accept'=>'Application/JSON;charset=utf-8', 'Calculaterelevancescore'=>'false', 'Content-Type'=>'TEXT/RAW', 'Docrdfaccessible'=>'false', 'Enablemetadatatype'=>'SocialTags', 'Host'=>'api.opencalais.com:80', 'Omitoutputtingoriginaltext'=>'TRUE', 'Outputformat'=>'Application/JSON'}).
+      stub_request(:post, "https://api.thomsonreuters.com/permid/calais").
+        with(headers: {'Accept'=>'application/json;charset=utf-8', 'Content-Type'=>'text/raw', 'Host'=>'api.thomsonreuters.com:443', 'Outputformat'=>'application/json', 'X-Ag-Access-Token'=>'opencaliasapikey', 'X-Calais-Language'=>'English'}).
```

Comment:
Line is too long. [243/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.